crates/store/src/state.rs

```diff
@@ -451,6 +451,51 @@ impl State {
         Ok((state_sync, delta))
     }
 
+    /// Loads data to synchronize a client's notes.
+    ///
+    /// The client's request contains a list of tag prefixes, this method will return the first
+    /// block with a matching tag, or the chain tip. All the other values are filter based on this
+    /// block range.
+    ///
+    /// # Arguments
+    ///
+    /// - `block_num`: The last block *know* by the client, updates start from the next block.
+    /// - `note_tag_prefixes`: Only the 16 high bits of the tags the client is interested in, result
+    ///   will include notes with matching prefixes, the first block with a matching note determines
+    ///   the block range.
```

Comment:
typo: `know` -> `known`.

Also the description needs to be updated to mention full tags rather than 16-bit prefixes.